pkg/network/node/egressip.go

```diff
@@ -85,34 +88,50 @@ func (eip *egressIPWatcher) Start(osdnInformers osdninformers.SharedInformerFact
 	return nil
 }
 
-func (eip *egressIPWatcher) Synced() {
+func (eip *egressIPWatcher) SyncIPAssignments() error {
 	link, _, err := GetLinkDetails(eip.localIP)
 	if err != nil {
 		// shouldn't happen, but obviously there's nothing to clean up...
-		return
+		return err
```

Comment:
comment should be updated, since it implies this will be a no-op, but it's not now